react/components/LoggedOutCTA/components/UI/index.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+import constants from 'react/styles/constants';
+import Text from 'react/components/UI/Text';
+import { GenericButtonLink } from 'react/components/UI/GenericButton';
+
+const Copy = styled(Text).attrs({
+  f: 5,
+  py: 3,
+  pr: 4,
+})``;
+
+const Button = styled(GenericButtonLink).attrs({
+  mx: 3,
+})`
+  ${constants.media.mobile`
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+  `}
+
+`;
+
+const Container = styled.div`
+  display: flex;
+  align-content: center;
+  justify-content: center;
+`;
```

Comment:
What's with the separate file for all of these things anyway. None of these components really require you to do a styled component custom for it. If that's easier for you then fine but it's much faster IMO to just inline these props on the components themselves `<Text f={5} py={3} pr={4}>`...